### ISSUE TITLE
Include required npm@v2 in readme

### ISSUE THREAD

**Issue by ulpian** [vdemedes/mongorito#105]
The package has issued when using latest release ofd npm@v3 because of mongoskin which is a dependency of monk (a package used in Mongorito);

```
npm WARN peerDependencies The peer dependency mongodb@~1.4 included from mongoskin will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```

Just thought it may be useful for some devs trying to install on v3 and may not know why it is crashing, my experience is that not a lot of devs read npm output :disappointed:

**Comment by vdemedes** [comment]
I think it's worth to make it work with npm@3 as well, instead of mentioning it's not supported, it's gaining "market share" quick! Would you be interested in doing a PR?

**Comment by ulpian** [comment]
I'll issue a PR for this.

**Comment by vdemedes** [comment]
Thanks, great!

**Comment by ulpian** [comment]
Very conveniently I am not able to replicate the issue now; I tried both mongorito `v2` and `v1.4.6` on npm versions `v2.15.*`, `v3.8.6`, `v3.8.8`, `v3.9.1` and I can see no issues.

Note that a lot of patches have been done on npm (https://github.com/npm/npm/releases) to fix different dependency issues and I guess somewhere on their issues so has mongodb, too many issues to trawl through (https://jira.mongodb.org/browse/SERVER-5177?jql=project%20%3D%20SERVER%20AND%20fixVersion%20in%20(2.0.4%2C%202.0.5)%20ORDER%20BY%20status%20DESC%2C%20priority%20DESC).

I will close this issue as I don't think anyone else willl have this issue, i believe it has been solved by the dependencies themselves.
